myriad-scheduler/src/main/java/org/apache/myriad/MyriadModule.java

```diff
@@ -116,7 +112,7 @@ protected void configure() {
         }
       } else {
         //TODO (kjyost) Confirm if this else statement and logic should still be here
```

Comment:
Recommend removal of kjyost comment here, appears this if...else statement is required